pkl/private/pkl_project.bzl

```
@@ -45,6 +45,24 @@ def parse_pkl_project_deps_json(pkl_deps_json_string_path):
 
     return seen_remote_pkl_packages.values()
 
+def convert_dict_to_options(option_name, items_dict):
+    """Converts the dictioinary to a list with each "key=value" pair preceded by the option name (e.g. --my-option).
+
+    Args:
+        option_name (str): The name of the option, typically in the form of `--option` or `-o`.
+        items_dict (dict[str, str]): A dictionary of items where:
+            - **Keys** are the names of the item (e.g., "key1", "key2").
+            - **Values** are the corresponding values for each item (e.g., "value1", "value2").
+    Returns:
+        A list of strings containing PKL CLI options.
+    """
+
+    list = []
+    for key, value in items_dict.items():
+        list.append(option_name)
+        list.append("{key}={value}".format(key = key, value = value))
```

```suggestion
        list.append("'{key}'='{value}'".format(key = key, value = value))
```
See remark about test coverage

---

I think adding quotation marks around key or value could be decided by the user based on thier needs
For example, a value like `I'm good` will break the evaluation command, as we surround it with single quotation marks

but if we left the decision to the user, it could be passed as `"I'm good"` and evaluation command would be happy

What you think?